packages/app/src/app/pages/Dashboard/Content/index.js

```diff
@@ -3,6 +3,7 @@ import { Route, Switch, Redirect, withRouter } from 'react-router-dom';
 
 import RecentSandboxes from './routes/RecentSandboxes';
 import PathedSandboxes from './routes/PathedSandboxes';
+import { PathedTemplates } from './routes/PathedTemplates';
```

Comment:
We can just call this "Templates". We used `PathedSandboxes` as name because we were using folders for sandboxes to organize them, we don't have this functionality for templates.